crates/bevy_ecs/src/entity/mod.rs

```diff
@@ -50,6 +50,9 @@ pub struct Entity {
     pub(crate) id: u32,
 }
 
+// TODO: Optimize memory usage of Option<Entity>
```

Comment:
Does this work yet?

Reply:
No I think I added this before I knew about the Entity niching PR. I'll remove this soon.